audio-worklet/src/bitcrusher-script-processor.js

```diff
@@ -49,6 +49,8 @@ class Bitcrusher {
     this.input.connect(this.node_).connect(this.output);
 
     // Index and previousSample defined as globals to handle block transitions.
+    // PreviousSample will be input[0] at specified bit depth on the first call
+    // to onaudioprocess since 0 % x == 0 
     this.index_ = 0;
     this.previousSample_;
```

Comment:
Seems to me that it's pretty harmless to initialize `previousSample_` to zero.  Then you don't have to comment on what it would be initialized to.